appium/webdriver/extensions/search_context/android.py

```diff
@@ -15,16 +15,21 @@
 # pylint: disable=abstract-method
 
 import json
+from typing import List
 
 from appium.webdriver.common.mobileby import MobileBy
 
 from .base_search_context import BaseSearchContext
 
+if False:
+    from appium.webdriver.webelement import WebElement  # For mypy
+
 
 class AndroidSearchContext(BaseSearchContext):
     """Define search context for Android"""
 
-    def find_element_by_android_data_matcher(self, name=None, args=None, className=None):
+    def find_element_by_android_data_matcher(
+            self, name: str = None, args: str = None, className: str = None):  # -> WebElement:
```

Comment:
why the retuned type is commented out?

Reply:
Because unit test is failed due to cyclic import.
Will fix later.